controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -995,8 +996,20 @@ bool master_thread::handle_cmdu_1905_ack_message(const std::string &src_mac,
                                                  ieee1905_1::CmduMessageRx &cmdu_rx)
 {
     auto mid = cmdu_rx.getMessageId();
+
+    // extract error code, if any
+    std::stringstream errorSS;
+    auto error_tlv = cmdu_rx.getClass<wfa_map::tlvErrorCode>();
+    if (error_tlv) {
+        errorSS << "0x" << error_tlv->reason_code();
+    } else {
+        errorSS << "no error";
+    }
+
+    LOG(DEBUG) << "Received ACK_MESSAGE, mid=" << std::hex << int(mid)
+               << " tlv error code: " << errorSS.str();
```

Comment:
I prefer not to. The reason is this:
It is most likely that in some future, when error flows are handled, we would like to do _more_ than just printing the error. e.g. adding some counters, taking recovery actions, etc.
Current code construction supports better these future changes. It would be easier for the developer to construct a code that behaves differently upon success and failure.
(there is always the option that the entire section will be rewritten. Nevertheless, current schema better reveals the options).